src/neural/encoder.h

```diff
@@ -46,6 +46,7 @@ int TransformForPosition(pblczero::NetworkFormat::InputFormat input_format,
 // Encodes the last position in history for the neural network request.
 InputPlanes EncodePositionForNN(
     pblczero::NetworkFormat::InputFormat input_format,
```

Comment:
Why not pass the `NetworkCapabilities` structure directly?

Reply:
A similar thought crossed my mind.  Also passing NetworkFormat to NetworkCapabilities instead of the 3 format properties.